packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/components/Label/Label.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { SpacedText } from '@zooniverse/react-components'
+import getDataSeriesColor from '../../../../../../helpers/getDataSeriesColor'
+import getDataSeriesSymbol from '../../../../../../helpers/getDataSeriesSymbol'
```

Comment:
Is this the quickest way to access this? I thought the monorepo allowed accessing from the root, so maybe it could be `import getDataSeriesSymbol from 'helpers/getDataSeriesSymbol'`. Either way, this is minor.

Reply:
This is a bit confusing unfortunately. The aliased `helpers` path is for the very top level `helpers` folder. The sub-folder for subject viewers also has a `helpers` folder which is what is being used here. The goal is to ideally break off the subject viewers into its own package for eventual reusability with Talk if we ever get around to redoing that (and maybe elsewhere?). I could alias these as `subject-viewer-helpers` for now to make this more clear.

Reply:
Actually, we already have `@viewers` aliased, so I'll use that